### PR TITLE
Add App Configuration information

### PR DESCRIPTION
**Pull request by AlexanderSher**

### CODE REVIEW

File: releases/2020-07/dotnet.md [hunk 1]
```diff
@@ -65,6 +65,10 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
 #### New features
 - `HttpPipeline.CreateClientRequestIdScope` method to allow setting client request id on outgoing requests.
 
+### App Configuration [Changelog](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/appconfiguration/Azure.Data.AppConfiguration/CHANGELOG.md)
```
Comment (ShivangiReja) [inline releases/2020-07/dotnet.md:68]: Alphabetically this will go first, before `Core`.

File: releases/2020-07/dotnet.md [hunk 2]
```diff
@@ -65,6 +65,10 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
...
+
+- Update the tag list for the AzConfig package
```
Comment (ShivangiReja) [inline releases/2020-07/dotnet.md:70]: Can we move this description under any of this?

```
#### Breaking Changes
 - Breaking changes
                
#### New Features
 - New features            

#### Key Bug Fixes
 - Notable bug fixes
```

File: releases/2020-07/dotnet.md [hunk 3]
```diff
@@ -15,7 +15,7 @@ The Azure SDK team is pleased to announce our July 2020 client library releases.
 
 #### Updates
 
-- _Add packages_
+- App Configuration
```
Comment (ShivangiReja) [inline releases/2020-07/dotnet.md:18]: Can we also add the installation instruction here: https://github.com/Azure/azure-sdk/blob/AzConfigUpdate/releases/2020-07/dotnet.md#installation-instructions in alphabalical order?

File: releases/2020-07/dotnet.md (hunk 2 above)
Comment (AlexanderSher) [reply releases/2020-07/dotnet.md:70]: No, the change doesn't fall into any of these categories

File: releases/2020-07/dotnet.md (hunk 3 above)
Comment (AlexanderSher) [reply releases/2020-07/dotnet.md:18]: Fixed

File: releases/2020-07/dotnet.md (hunk 1 above)
Comment (AlexanderSher) [reply releases/2020-07/dotnet.md]: Fixed

File: releases/2020-07/dotnet.md [hunk 4]
```diff
@@ -34,6 +34,8 @@ The Azure SDK team is pleased to announce our July 2020 client library releases.
 To install any of our packages, please search for them via `Manage NuGet Packages...` in Visual Studio (with `Include prerelease` checked) or copy these commands into your terminal:
 
 ```bash
+ $> dotnet add package Azure.Data.AppConfiguration
+
  $> dotnet add package Azure.AI.TextAnalytics
```
Comment (ShivangiReja) [inline releases/2020-07/dotnet.md:39]:
```suggestion
 $> dotnet add package Azure.AI.TextAnalytics

 $> dotnet add package Azure.Data.AppConfiguration
```

File: releases/2020-07/dotnet.md [hunk 5]
```diff
@@ -60,6 +62,10 @@ If you have a bug or feature request for one of the libraries, please [file an i
 
 Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here are some of the highlights:
 
+### App Configuration [Changelog](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/appconfiguration/Azure.Data.AppConfiguration/CHANGELOG.md)
```
Comment (ShivangiReja) [inline releases/2020-07/dotnet.md:65]: Can we point the changelog to the particular release date? i.e. https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/appconfiguration/Azure.Data.AppConfiguration/CHANGELOG.md#101-2020-07-07 Or we can use released tag for the changelog link.

File: releases/2020-07/dotnet.md (hunk 4 above)
Comment (AlexanderSher) [reply releases/2020-07/dotnet.md]: Fixed

**Pull request merged**